director/director_api.go

```diff
@@ -78,7 +79,7 @@ func CreateDirectorSDToken() (string, error) {
 	tokenExpireTime := param.Monitoring_TokenExpiresIn.GetDuration()
 
 	tok, err := jwt.NewBuilder().
-		Claim("scope", "pelican.directorSD").
+		Claim("scope", utils.Pelican_DirectorServiceDiscovery.String()).
```

Comment:
Here (and throughout PR) -- from the names, it's not clear that these are token scopes.  Options could be:
- Rename the package called `token_scopes` or similar.
- Add a prefix to the variable name (`TokenScope_Pelican_DirectorServiceDiscovery`).

I think I'd lean toward the package rename.